app/controllers/mixins/generic_show_mixin.rb

```diff
@@ -66,26 +66,35 @@ def display_nested_list(display)
       respond_to?(nested_list_method_name(display).to_sym) ? nested_list_call(display) : display_nested_generic(display)
     end
 
+    def prepare_breadcrumb(title)
+      title = title.is_a?(Hash) ? _("(All %{resource_name})") % {:resource_name => ui_lookup(:tables => title[:table])} : title
```

Comment:
I don't like this at all.

 * do we really need to pass a hash?
 * if we pass a hash do we need to confuse people by passing in `:table` and then using it as `:tables` that is counter-intuitive and goes against the pattern or `ui_lookup`.

And if you would be changing the method to accept variants of input data I'd go for something like:
```
nested_list(model, options)
```
and call it either 
```
nested_list(ManageIQ::Providers::CloudManager::Vm, :title => 'String')
```
or
```
nested_list(ManageIQ::Providers::CloudManager::Vm, :tables => 'table_name')
```

You should utilize `grep` to check if other forms are going to be needed.

I am not saying it must be like that, I am just throwing ideas. But I am quite sure the solution your are offering needs some rethinking.

Also, please, consider that some differences between the form of breadcrumbs might me just a mess that need's a cleanup and unification.